postinstall/upgrade_many.py

```diff
@@ -262,11 +295,12 @@ def ssh_unchecked_call(*args):
                     ssh_call("weka", "local", "start", container_name,)
                 raise
 
+        set_agent_version_if_all_container_are_in_target_version()
 
         upgraded_hosts += 1
         if not skip_health_checks:
         # We first want to see the cluster as unhealthy before we wait for it to become healthy
-            if not skip_unhealthy_checks:
+            if not skip_unhealthy_checks and container_has_drives():
```

Comment:
healthy cluster sounds like something that is relevant to all containers and not just drives.
rename wait_for_unhealthy_cluster to wait_for_rebuild_to_start or something

Reply:
sure